components/global_header/left_controls/product_menu/product_menu_list/product_menu_list.tsx

```diff
@@ -14,7 +14,7 @@ import TeamPermissionGate from 'components/permissions_gates/team_permission_gat
 import MarketplaceModal from 'components/plugin_marketplace';
 import Menu from 'components/widgets/menu/menu';
 import {ModalIdentifiers} from 'utils/constants';
-import {useSafeUrl} from 'utils/url';
+import {makeUrlSafe} from 'utils/url';
```

Comment:
This one's kind of a funny one. The plugin assumes anything named `useX` is a hook. I spent a bunch of time trying to fix the usage of this one before realizing it wasn't actually a hook. We'll just have to be aware down the line that 'useX' is a name reserved for hooks.